sql/expression/convert.go

```diff
@@ -120,46 +131,51 @@ func (c *Convert) Eval(ctx *sql.Context, row sql.Row) (interface{}, error) {
 func convertValue(val interface{}, castTo string) (interface{}, error) {
```

Comment:
What's with the change in error handling in this function? You changed many error handling blocks to return nil on encountering an error. This is either a mistake or needs to be carefully documented.